.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Just like NASA going to the moon, it's always good to have a checklist when
+creating changes.
+The following items are listed to help you create a great Pull Request: -->
+## Checklist
+- [ ] All new and existing **tests are passing**
+  - [ ] Python tests using `make test flakes` are passing
+  - [ ] Javascript unit tests using `npm run test-unit` are passing
+  - [ ] Javascript integration tests using `npm run test-integration` are passing
+- [ ] I have added tests to cover my changes
+- [ ] My change requires a change to the **documentation**.
+  - [ ] I have updated the documentation accordingly.
+  - [ ] I have added an entry to `CHANGES` because this is a user-facing change or an important bugfix
+- [ ] I have written **proper commit message(s)**
+      <!-- Title under 50 characters, good explanations, affected part of Isso
+      mentioned in title, e.g. "isso: css: Increase font-size on buttons"
```

Comment:
Why "isso: "? isn't that implied for all changes to this repo?

Reply:
I agree, it should just be `css: Some change` or `js: Some client change` for example.

Reply:
I was thinking of the client css (`isso: css: ...`) as opposed to the docs css (`docs: css`).
But I agree with both of you and have pushed with a better example.